src/libdrakvuf/libdrakvuf.h

```diff
@@ -414,6 +414,11 @@ int drakvuf_event_fd_add(drakvuf_t drakvuf,
 int drakvuf_event_fd_remove(drakvuf_t drakvuf,
                             int fd);
 
+unicode_string_t* read_wchar_array(vmi_instance_t vmi, const access_context_t* ctx, size_t length);
```

Comment:
These functions will need to be prefixed with `drakvuf` like all the other functions in this header. Also, comment them to explain what they are for

Reply:
Ok

Reply:
Done

Reply:
The functions stilldon't seem to be properly prefixed here.